client/blocks/login/style.scss

```diff
@@ -76,6 +76,13 @@
 .login__form-signup-link {
 	font-size: 13px;
 	margin-top: 20px;
+
+	&.disabled {
```

Comment:
In this file there is a same syntax for `&.is-error { ... }`, so I guess that's fine?
@scruffian what's your take on that?